asilib/asi/trex.py

```diff
@@ -510,6 +517,7 @@ def trex_nir(
     redownload: bool = False,
     missing_ok: bool = True,
     load_images: bool = True,
+    acknowledgements_always: bool = False,
```

Comment:
This PR did not pass the tests because the function argument name is "acknowledgements_always" while the local variable name is "acknowledge_always". That's an easy fix!